client/app/pages/queries-list/queries-list.html

```diff
@@ -97,8 +97,9 @@
               <td class="table-main-title">
                 <a href="queries/{{query.id}}">{{query.name}}</a>
                 <br>
-                <span class="label label-default" ng-if="query.is_draft">Unpublished</span>
-                <tags-control item="query"></tags-control>
+                <span class="maxwidth300">
```

Comment:
What is this for?

Reply:
Without it the queries list will go long into the right sidebar.

Reply:
There is further discussion in #2829